src/main/drivers/accgyro/accgyro_spi_bmi160.c

```diff
@@ -164,8 +164,8 @@ static int32_t BMI160_Config(const extDevice_t *dev)
     spiWriteReg(dev, BMI160_REG_ACC_CONF, 0x20 | BMI160_ODR_800_Hz);
     delay(1);
 
-    // Set gyr_bwp = 0b010 so only the first filter stage is used
-    spiWriteReg(dev, BMI160_REG_GYR_CONF, 0x20 | BMI160_ODR_3200_Hz);
+    // Set gyr_bwp = 0b000 so only the OSR4 mode is used
+    spiWriteReg(dev, BMI160_REG_GYR_CONF, 0x00 | BMI160_ODR_3200_Hz);
```

Comment:
This is already against the original development intent. At the time, although we wanted to find a good replacement for the MPU6000 gyroscope, we didn't want to make it similar to the MPU6000 in every way. If a certain gyroscope has better signal performance under certain conditions, why do extra work to weaken its performance for better "management"?

Reply:
@DusKing1 , a user with BMI gyro has to apply a BMI preset to make this gyro work nicely. So we have two options:
1. leave it as is.
2. try OSR4 mode - i guess then we don't need BMI filters presets?
Wondering which way provides better performance and how would you measure this performance to compare?

Reply:
@etracer65 as you have worked on this please comment / review

Reply:
Assuming that the signal quality of the gyroscope itself is not affected by the OSR4 register, only the SNR of the output signal changes. Then as we are still using the linear filtering method, Heisenberg principle still applies: if we wanna achieve same signal quality, then the amount of SLPF+DLPF should somehow be the same.
But how can me make sure they are the same considering the DLPF model inside the BMI270 is developed and own by Bosch and we have no idea what it is? That's the extra effort and work burden for other devs.
Our original idea at the beginning of development was to minimize the use of DLPF (rather than disable it completely) without causing extremely bad gyroscope signals, to minimize the signal delay due to DLPF, so that we can be in betaflight later filtering it in a controlled method, although this may result in some extra LPF settings, is in line with what betaflight was developed for: to bring the best possible flight performance to the user, instead of the easiest.

Reply:
In order to have sufficient attenuation of noise above 350Hz or so additional software filtering has to be used in BF. The total delay of DLPF + SLPF may actually be more than simply setting OSR4 mode and turning off the SLPF.

As an example I've produced attenuation and delay plots for OSR4 mode vs DLPF+SLPF assuming all filters are biquads:

![image](https://user-images.githubusercontent.com/41840611/153742635-9359e2ff-dc38-473a-897f-fff44d24f045.png)

Notice that even though both filters have equal attenuation in the region of interest (up to 500Hz) because the DLPF+SLPF has more attenuation at higher frequencies it ends up with more delay.

Reply:
> The total delay of DLPF + SLPF may actually be more than simply setting OSR4 mode and turning off the SLPF.
Why turn off?
> As an example I've produced attenuation and delay plots for OSR4 mode vs DLPF+SLPF assuming all filters are biquads:
Why all filters are all biquads?

Reply:
With OSR4 mode on you would turn off the SLPF to reduce delay because it is no longer needed.
I used biquads because you told me that's what you thought was used inside the gyro on discord.

Reply:
So then? Where's the result of the no betaflight LPF spectrum chart to show how it looks like?

Reply:
PSD plots are in the first post. 👍

Reply:
Do you mean you just fly on the raw gyro signal even without dyn-notches or rpm-filters?

Reply:
No I used gyro scaled debug mode.